### ISSUE TITLE
[UX] Update description text for block class field

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#3608]
While doing our training yesterday, I noticed that the description on the block class field did not mention that we need to separate classes with spaces. Instead, it reads `Additional classes to be added to this block.` ...which doesn't add any value beyond the field label: `Block classes`. 

<img width="654" alt="Screen Shot 2019-03-21 at 5 48 42 PM" src="https://user-images.githubusercontent.com/397895/54789768-9b8e8e00-4c01-11e9-97f3-0af8fda37895.png">

How about we change this description text to: `Separate multiple classes with spaces`

---
PR by @Plochem: https://github.com/backdrop/backdrop/pull/2598
~follow-up PR by @klonos: https://github.com/backdrop/backdrop/pull/2631~
Follow-up PR: https://github.com/backdrop/backdrop/pull/2876

**Comment by arjunk2004** [comment]
Sorry, I accidentally closed it

**Comment by klonos** [comment]
A few comments:

1. We should specify that these are **CSS** classes.

2. I think that the word "additional" should remain, otherwise we might create the impression that the classes specified here will be the only ones added to the block. In Views, we specifically have a "Add default classes" option, which is not available here:

    <img width="565" alt="Screen Shot 2019-04-23 at 6 14 54 am" src="https://user-images.githubusercontent.com/2423362/56526177-53abaf80-658f-11e9-9fd8-7ba71d3fbfcc.png">

    Perhaps changing the field label to `Additional CSS classes` would work.

PS: In general, these use cases is why I would like us to have a dedicated `css_classes` field introduced in the Form API. It should have a default description, so that we don't have to be re-inventing things each time we add such a field + this would allow consistency for this description throughout the UI (unless we need to specifically override it in certain cases) + would also allow translating the description only once (instead of having separate translations for each "mutation").

**Comment by jenlampton** [comment]
> PS: In general, these use cases is why I would like us to have a dedicated css_classes field introduced in the Form API.

Is there an issue for this yet? I like the special validation we'd get, and consistent description text.

**Comment by klonos** [comment]
>Is there an issue for this yet?

Not until someone else was also interested in this: #3702 😉

**Comment by quicksketch** [comment]
I merged https://github.com/backdrop/backdrop/pull/2598 before I saw that there were other things needed here. I'm reopening this to iterate further, but since it looks like we want to keep the description the same(?) but now change the field label, I don't think there's a need to revert.

Thanks @Plochem for the PR!

Leaving this open for further changes.

**Comment by klonos** [comment]
...the PR for #3026 accidentally included [this change](https://github.com/backdrop/backdrop/pull/2605/files#diff-c328db204c966e5c9672fb00cf98b810R40).

**Comment by quicksketch** [comment]
I did see that, I merged https://github.com/backdrop/backdrop/pull/2598 first, which had that same change. No harm done and I don't think there is anything that needs reverting(?) though this issue still has some further work to consider.

**Comment by klonos** [comment]
> ...I don't think there is anything that needs reverting(?)

Nope, not pedantic about it either. All good 👍

> Next steps for this issue are:
> 
> - Change "Block classes" title text to `Additional CSS Classes`

That, plus adding `These CSS classes will be added to the default ones for this block.` before the `Separate multiple classes with spaces.` bit: https://github.com/backdrop/backdrop/pull/2631

Let me know if you think that it is too "wordy" now:

<img width="587" alt="Screen Shot 2019-04-30 at 8 06 46 am" src="https://user-images.githubusercontent.com/2423362/56930206-43f81200-6b1f-11e9-989d-6bf140e45c7c.png">

**Comment by olafgrabienski** [comment]
> Let me know if you think that it is too "wordy" now

Hard to say, but let's try to improve:

`Separate multiple classes with spaces. They will be added to the default classes for this block.`

Doing so, we'd

- put the more important information (for a content editor) at the beginning,
- avoid `the default ones` (sound a bit awkward to me),
- have 10 characters less.

Even shorter without `to this block` (not sure):

`Separate multiple classes with spaces. They will be added to the default classes.`

**Comment by jenlampton** [comment]
We don't need the second sentence if the lable starts with 'Additional'.

**Comment by BWPanda** [comment]
I agree with @jenlampton that we don't need to re-word the description from what it is now. So here's a new PR that just changes the title: https://github.com/backdrop/backdrop/pull/2876

**Comment by klonos** [comment]
Thanks @BWPanda 👍 RTBC 🙂

**Comment by quicksketch** [comment]
Thanks folks! Merged https://github.com/backdrop/backdrop/pull/2876 into 1.x and 1.13.x.
